src/main/kotlin/apply/domain/evaluationtarget/EvaluationAnswer.kt

```diff
@@ -6,13 +6,8 @@ import javax.persistence.Embeddable
 @Embeddable
 class EvaluationAnswer(
     @Column(nullable = false)
-    private var score: Int = 0,
+    internal val score: Int = 0,
```

Comment:
`internal`는 그다지 의미가 없으니 통일성을 위해 public으로 바꾸면 어떨까요?

```suggestion
    val score: Int = 0,
```